scenarios/centos-9/multinode-ci.yml

```diff
@@ -4,6 +4,12 @@ cifmw_use_libvirt: false
 
 cifmw_openshift_setup_skip_internal_registry_tls_verify: true
 
+pre_infra:
```

Comment:
We can't use `pre_infra` here as OCP isn't available at that stage. 

`post_infra` looks to be the correct hook to use but I didn't test.

You can see CI failing due to this[1]:
```
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Could not find or access '/home/zuul/.crc/machines/crc/kubeconfig' on the Ansible Controller.\nIf you are using a module and expect the file to exist on the remote, see the remote_src option"}
```
[1] https://logserver.rdoproject.org/82/1482/43267db1d1546643bcf8d8f5f0a5fc8cc69f8f2d/github-check/podified-multinode-edpm-deployment-crc/a1df9e4/controller/ci-framework-data/logs/ci_script_000_run_disable_enable_red_hat.log

Reply:
Done!

Reply:
Thank you @pablintino @lewisdenny for the review. Since OCP is deployed after pre_infra hook and cert_manager role iscalled before post_infra. There is no way to run this workaround
as a hook so we need to include in the cert_manager role itself. I have updated the same.

Reply:
Nice, let's see how CI likes it. Thanks for adding the comment too :)